src/app/containers/Image/index.test.jsx

```diff
@@ -87,9 +88,12 @@ describe('Image', () => {
     );
 
     it('should render a lazyload container instead of an image if the image is after the 3rd block', () => {
-      // Render using enzyme to capture noscript contents
-      const container = render(<ImageContainer position={[4]} {...data} />);
-      expect(container).toMatchSnapshot();
```

Comment:
woops did I delete this snapshot?! maybe add this back in

Reply:
😱

Reply:
With the additional assertions, it can probably remain deleted? 😄

Reply:
I actually removed it by accident 🙈 

Added back in now.